{
    "assignee": null,
    "assignees": [],
    "author_association": "NONE",
    "body": "When building a file called `cython.pyx` via \r\n\r\n`cythonize cython.pyx` \r\n\r\nI get an error message saying: \"ValueError: cython is a special module, cannot be used as a module name\" which is a Good Thing.\r\n\r\nHowever, using `cython cython.pyx`  doesn't produce any errors but a cython.c file - one which at least at Linux cannot be compiled.\r\n\r\nGiven that `cython` - tool is advertized in documentation (http://cython.readthedocs.io/en/latest/src/reference/compilation.html#compiling-from-the-command-line) and that it is not unusual for somebody new to Cython to call her/his pyx-file \"cython.pyx\", please consider to produce the same error message also with `cython`-tool.",
    "closed_at": null,
    "comment_data": [
        {
            "author_association": "MEMBER",
            "body": "PR welcome\n",
            "created_at": "2018-06-26T15:07:46Z",
            "html_url": "https://github.com/cython/cython/issues/2422#issuecomment-400344691",
            "id": 400344691,
            "issue_url": "https://api.github.com/repos/cython/cython/issues/2422",
            "node_id": "MDEyOklzc3VlQ29tbWVudDQwMDM0NDY5MQ==",
            "updated_at": "2018-06-26T15:07:46Z",
            "url": "https://api.github.com/repos/cython/cython/issues/comments/400344691",
            "user": {
                "avatar_url": "https://avatars0.githubusercontent.com/u/491659?v=4",
                "events_url": "https://api.github.com/users/scoder/events{/privacy}",
                "followers_url": "https://api.github.com/users/scoder/followers",
                "following_url": "https://api.github.com/users/scoder/following{/other_user}",
                "gists_url": "https://api.github.com/users/scoder/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/scoder",
                "id": 491659,
                "login": "scoder",
                "node_id": "MDQ6VXNlcjQ5MTY1OQ==",
                "organizations_url": "https://api.github.com/users/scoder/orgs",
                "received_events_url": "https://api.github.com/users/scoder/received_events",
                "repos_url": "https://api.github.com/users/scoder/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/scoder/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/scoder/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/scoder"
            }
        }
    ],
    "comments": 1,
    "comments_url": "https://api.github.com/repos/cython/cython/issues/2422/comments",
    "created_at": "2018-06-26T06:24:42Z",
    "event_data": [],
    "events_url": "https://api.github.com/repos/cython/cython/issues/2422/events",
    "html_url": "https://github.com/cython/cython/issues/2422",
    "id": 335674985,
    "labels": [],
    "labels_url": "https://api.github.com/repos/cython/cython/issues/2422/labels{/name}",
    "locked": false,
    "milestone": null,
    "node_id": "MDU6SXNzdWUzMzU2NzQ5ODU=",
    "number": 2422,
    "repository_url": "https://api.github.com/repos/cython/cython",
    "state": "open",
    "title": "cython-tool might produce an error for cython.pyx, the same way cythonize does",
    "updated_at": "2018-06-26T15:07:46Z",
    "url": "https://api.github.com/repos/cython/cython/issues/2422",
    "user": {
        "avatar_url": "https://avatars2.githubusercontent.com/u/17513912?v=4",
        "events_url": "https://api.github.com/users/realead/events{/privacy}",
        "followers_url": "https://api.github.com/users/realead/followers",
        "following_url": "https://api.github.com/users/realead/following{/other_user}",
        "gists_url": "https://api.github.com/users/realead/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/realead",
        "id": 17513912,
        "login": "realead",
        "node_id": "MDQ6VXNlcjE3NTEzOTEy",
        "organizations_url": "https://api.github.com/users/realead/orgs",
        "received_events_url": "https://api.github.com/users/realead/received_events",
        "repos_url": "https://api.github.com/users/realead/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/realead/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/realead/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/realead"
    }
}